src/components/ClusterDeployment/ClusterDeploymentWizardFooter.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Alerts from '../ui/Alerts';
+import { useAlerts } from '../AlertsContextProvider';
+import { WizardFooter } from '../ui';
+import { WizardFooterGenericProps } from '../ui/WizardFooter';
+import { Alert, AlertGroup, AlertVariant } from '@patternfly/react-core';
+import { CLUSTER_DEPLOYMENT_FIELD_LABELS } from './constants';
+
+type ClusterDeploymentWizardFooterProps = WizardFooterGenericProps & {
+  // additionalActions?: React.ReactNode;
+  errorFields?: string[];
+};
+
+const ValidationSection = ({ errorFields = [] }: { errorFields?: string[] }) => {
```

Comment:
lets use React.FC, same for `ClusterDeploymentWizardFooter`

```suggestion
const ValidationSection: React.FC<ValidationSectionProps> = ({ errorFields }) => {
```

Reply:
done